hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -176,8 +179,16 @@ spec =
                 { initials = newInitials
                 , commits = newCommits
                 } = snd <$> observeCommit tx onChainState
-         in newInitials == (mkInitials <$> Prelude.tail inputs)
-              && newCommits == [(commitInput, commitOutput, commitDatum)]
+
+            noOverlap =
```

Comment:
Suggestion: More explicit name, like `commitedUtxoDoesNotOverlapWithInitials`? To understand what `noOverlap` meant I have had to expand the code in the Github interface.